### ISSUE TITLE
Notification during voice call - Background State

### ISSUE THREAD

**Issue by codoffer** [ZEGOCLOUD/zego_uikit_prebuilt_call_flutter#50]
- [x] I have read the [Getting Started](https://www.zegocloud.com/doc/callkit/Call%20Kit/main/Quick%20start/b26d509e) and [Getting Started(with call invitation)](https://www.zegocloud.com/doc/callkit/Call%20Kit/main/Quick%20start%20(with%20call%20invitation)/eb1d3c42)
- [x] I have already searched for the same problem

## Environment

| Technology            | Version       |
|-----------------------| ------------- |
| Flutter version      |       3.22.2      |
| Plugin version       |       ^4.12.1       |
| Android version      |    14          |
| iOS version          |        17.4      |
| macOS version        |              |
| Xcode version        |     15.4          |
| Google Chrome version |    125.0.6422.142          |

Device information: Realme 11 Pro Plus - Android 14 and iOS 12

## Description

**Expected behavior:** 

When an app is in the background state and there’s an ongoing voice call, it’s essential to have a clear indication for the active call.

**Current behavior:** 

If the user puts the application to the background stage during the active call, there is no indication of the active call.

## Steps to reproduce

Please check current voice call functionality with example repo.

**Comment by phluidaungyezaw** [comment]
any solution? @codoffer . I also facing same issue.

**Comment by yoer** [comment]
I don’t quite understand, but it means that if you are currently in the call (busy), and there are calls from other users at this time, do you need to get notify by event?

**Comment by priyankaarora-collab** [comment]
@yoer No, It means that if I'm on an active in-app call using ZEGOCLOUD and I close the app or move it to the background, there's no indication in the notification bar that the call is still active.

Ideally, there should be a persistent ongoing call notification (similar to WhatsApp) so that even when the app is in the background, the user knows there's an active in-app call and can return to or manage it from the notification.
